From 83da6daf033d3ea2153a4e01cebdf255beb0edb1 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 5 Sep 2017 13:58:21 +0800
Subject: [PATCH 2504/5242] MLK-16370-2 video: fbdev: dcss: remove YUV422
 support in dpr

commit  ebbf4a12a494c31de0eed9aa25f7cf2733dd37b1 from
https://source.codeaurora.org/external/imx/linux-imx.git

Return error value when meeting 'YUV422' pixel format,
since there is no code can handle this format, and this
can also avoid some build warning.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/imx_dcss.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/imx_dcss.c b/drivers/video/fbdev/mxc/imx_dcss.c
index 017f3d2..6285b8f 100644
--- a/drivers/video/fbdev/mxc/imx_dcss.c
+++ b/drivers/video/fbdev/mxc/imx_dcss.c
@@ -1488,7 +1488,7 @@ static int dcss_dpr_config(uint32_t dpr_ch, struct dcss_info *info)
 		}
 		break;
 	case 1:         /* TODO: YUV 1P */
-		break;
+		return -EINVAL;
 	case 2:         /* YUV 2P */
 		/* Two planes YUV format */
 		num_pix_y = dpr_pix_y_calc(0, input->height, input->tile_type);
-- 
1.7.9.5

